frontend/src/app/api/auth/[...nextauth]/config.ts

```diff
@@ -1,26 +1,63 @@
 import { AuthOptions } from "next-auth";
 import GoogleProvider from "next-auth/providers/google";
-
+import DiscordProvider from "next-auth/providers/discord";
 export const authOptions: AuthOptions = {
   providers: [
     GoogleProvider({
       clientId: process.env.NEXT_PUBLIC_GOOGLE_AUTH_CLIENT_ID as string,
       clientSecret: process.env.GOOGLE_AUTH_CLIENT_SECRET as string,
       checks: [],
     }),
+    DiscordProvider({
+      clientId: process.env.NEXT_PUBLIC_DISCORD_AUTH_CLIENT_ID as string,
```

Comment:
I don't seen any reason for the client id to be public, we should remove `NEXT_PUBLIC_` from the env name (same for Google above)

Reply:
You're right, I took this from supabase oauth providers